### DISCUSSION TITLE
Toggle sources(or items from the menu) based on filetype or LSP?

### DISCUSSION THREAD

**Discussion by sagbdev** [hrsh7th/nvim-cmp#686] [Q&A] [answered]
I don't know if this is a dumb question or not, I didn't find anything on the README related to it, but basically I use LuaSnip and friendly-snippets to get my snippets from, and it's not really a problem but whenever I use my actual setup with an HTML file I get two sources on the cmp menu, one from the LS in this case emmet_ls, and one from the snippets engine, which is a little bit annoying, is there any way of disabling a source/item from the cmp menu list whenever I enter an HTML file or if certain LS gets used? Or maybe with a keybind, like a toggle for the snippet engine to turn it on/off?

**Comment by dmitmel** [comment, marked as answer]
See #666, that one should help.

**Reply by sagbdev** [reply]
Thanks!
